resources/linux/debian/postinst.template

```diff
@@ -35,9 +35,18 @@ if [ "@@NAME@@" != "code-oss" ]; then
 	eval $(apt-config shell APT_TRUSTED_PARTS Dir::Etc::trustedparts/d)
 	CODE_TRUSTED_PART=${APT_TRUSTED_PARTS}microsoft.gpg
 
+	. /usr/share/debconf/confmodule
```

Comment:
Add check for file existence, ref https://github.com/microsoft/vscode/pull/218463/files#diff-b898935e3399366ac284fdcb5c51482e8649049b572319330b14eff726385266R19